keyboards/anavi/macropad8/keymaps/default/keymap.c

```diff
@@ -0,0 +1,68 @@
+#include QMK_KEYBOARD_H
+
+#define _MAIN 0
+#define _FN 1
+
+#define KC_X0 LT(_FN, KC_ESC)
+
+#ifdef RGBLIGHT_ENABLE
+// How long (in ms) to wait between animation steps for the rainbow mode
+const uint8_t RGBLED_RAINBOW_MOOD_INTERVALS[] PROGMEM = {60, 30, 15};
+// How long (in milliseconds) to wait between animation steps for each of the "Swirling rainbow" animations
+const uint8_t RGBLED_RAINBOW_SWIRL_INTERVALS[] PROGMEM = {20, 10, 4};
+#endif
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_MAIN] = LAYOUT(
+     LGUI(KC_D), KC_UP, LCTL(KC_C), LCTL(KC_V),
+     KC_LEFT, KC_DOWN, KC_RIGHT, MO(_FN)
+  ),
+
+  [_FN] = LAYOUT(
+     RGB_TOG, RGB_MOD, RGB_MODE_RAINBOW, RGB_MODE_SNAKE,
```

Comment:
If you wanted, you could do this: 
```suggestion
     RGB_TOG, RGB_MOD, RGB_M_R, RGB_M_SN,
```

Reply:
OK, I did it in the new version of the patch. Thanks, Leon